src/pacmanity.sh

```diff
@@ -1,51 +1,52 @@
 #!/bin/bash
-#
-# pacmanity generates a list of packages installed via pacman GitHub
+# Pacmanity
+# Keeps a list of installed packages in a Gist at your GitHub account.
 
-pacmanity(){
-    # Add Ruby to PATH
-    PATH="$(ruby -e 'print Gem.user_dir')/bin:$PATH"
-
-    # Load Config
-    [[ -r '/etc/pacmanity' ]] && source $pkgdir/etc/pacmanity
+pacmanity_install(){
+    echo "A list of installed packages will be automatically maintained"
+    echo "by Pacmanity in a private Gist at your GitHub account."
+
+    echo -e "\n- Step 1: Log in to Gist using your GitHub account:"
+    [[ -f ~/.gist ]] || gist --login
+    mkdir -p $pkgdir/root
+    cp -v ~/.gist $pkgdir/root/.gist
+
+    echo -e "\n- Step 2: Save list of currently installed packages to Gist:"
+    GIST_URL=$(
+        (
+            pacman -Qqen
+            echo
+            pacman -Qqem
+        ) | gist -p -f $HOSTNAME.pacmanity -d "$HOSTNAME: List of installed packages"
+    )
+
+    echo "GIST_ID=$GIST_URL" | sed 's/https:\/\/gist.github.com\///g' >> $pkgdir/etc/pacmanity
+
+    echo "An automatically mantained list of installed packages"
+    echo "has been successfully created in your GitHub Gist."
+    echo "Visit https://gist.github.com or the direct link below:"
+    echo "$GIST_URL"
+}
 
-    # Determine if Fresh Install is Needed
-    if [ -z "$GIST_ID" ]; then
-      echo -e "\nInfo: Gist backup for pacman-installed apps not setup."
-      echo -e "\nYou need to use your GitHub credentials to log into GitHub Gist."
+pacmanity_update(){
+    if (pacman -Qqen; echo; pacman -Qqem) | gist -u "$GIST_ID" -f $HOSTNAME.pacmanity; then
+        echo "Pacmanity: OK!"
```

Comment:
I know this is picky, but I wanted the message to still be reflective of what it is doing.  Like "Pacmanity: Updated package list gist."